marklogic-data-hub-central/ui/src/App.scss

```diff
@@ -1,258 +1,290 @@
 @font-face {
-  font-family: MLCustomFont;
-  src: url(./fonts/ML_IconFont_explore.ttf);
-  font-family: 'MLCustomFont'format('truetype');
-  font-weight: normal;
-  font-style: normal;
+    font-family: MLCustomFont;
```

Comment:
There are a lot of whitespace changes in here - wondering if those will cause conflicts when 5.4-develop is rebased on top of this, or are these whitespace changes already present on 5.4?

Reply:
I think these were due to tab space discrepancies between IDE's. I can look into reverting those changes.